### DISCUSSION TITLE
ramp_up_install

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/labview_book#16] [Announcements]
安装

https://lv.qizhen.xyz/ramp_up_install

**Comment by ruanqizhen** [comment]
之前博客上的相关文章，搬到这里：

## 在文件夹下直接创建新的 VI


    在 Windows Explore 中，鼠标右键点击某一文件夹的空白处，弹出的菜单中有“新建”一项。通过修改 Windows 的注册表，可以给这个新建列表添加一项，从而直接在文件夹下创建一个新的 VI。如图1所示。

![](http://byfiles.storage.msn.com/y1pPUK7j7lmY1WD2SxPznseE25mOUSHFAqQzbrF63Pse--FzHNBm8v8fIVLoIFnNDLJ)  
图1：在“新建”菜单中添加一项-创建VI

    下载 [](http://community.ni.com/examples/5728-windows-explor-4e2d76f463a5751f621065b0vi/create-new-vi.zip)[Create New VI.zip](http://decibel.ni.com/content/docs/DOC-1078) 文件，然后解压缩。运行里面的 Create Empty VI.reg，把它里面的内容导入到注册表。然后就会发现多出图1所示的新建 VI 的项目了。

    让 Windows 多一个新建项目，只要在注册表里添加上相关内容就行了。Create Empty VI.reg 文件中的 Data 数据其实就是新建出来的文件的内容。在这个例子中，他是一个空白的 LabVIEW 8.0 的 VI。我们可以改变 reg 文件中的数据，使得产生出来的 VI 有所不同。  
    比如，你希望自己的新 VI 总是使用蓝色背景的，有一个特殊的图标等等，只要改变 reg 文件中的数据就可以了。ZIP 包中还有一个 Create New Reg Data.vi，这个 VI 可以读入一个文件，把它转换成注册表中实用的 Data 数据。使用者可以自己先造一个自己喜欢的 VI 作为模板，然后利用Create New Reg Data.vi 为它创建出一个注册表数据，导入注册表。这样每次在文件夹下用鼠标右键创建出的 VI 就是模板 VI 的样子了。

**Reply by ruanqizhen** [reply]
之前博客下的留言：

军
2008年10月10日 15:48 编辑
好呀 可惜压缩包损坏无法打开！

回复
刘伟印
2008年08月15日 13:29 编辑
比较有意思，不过感觉实用性不大，即使快捷菜单创建了New VI要打开还是要启动LabVIEW环境啊

回复
竹贤
2008年01月12日 18:05 编辑
直接修改注册表也可，怎么改？

回复
竹贤
2008年01月11日 11:19 编辑
我觉得有用：有时需要搞一个新项目或新程序，省去了先打开VI环境，简化了操作。这与在桌面新建office的空白文档的功能是一样的，给使用者带来了便利。

回复
Qizhen
2007年05月31日 08:33 编辑
自己造个空的 7.1版 VI，用压缩包里的 VI 转换成注册表数据就可以了。

回复
雨夜
2007年05月29日 19:47 编辑
请问具体是怎样实现的，我自己也想做一个，我用的是lv7.1你的8.0我用不了！

回复
杰
2007年05月20日 22:36 编辑
这个好像不是太有用，嘿嘿。
干活的时候，总是在LV环境中的，如果需要特定的VI，有模版的话，打开这个VI，然后SAVE AS。
要在资源管理器里面直接新建的话，第一没有必要，因为程序总是要自己写的。没有LV环境，建新VI有啥用啊？第二，自己改下注册表就OK。
呵呵，随便说说。

回复
